From dfe25a73ca1589737a7247a2f89ae00a7683b238 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Mon, 13 Mar 2017 22:49:18 +0100
Subject: [PATCH 5/9] SystemUI: Dismiss keyguard on boot if disabled by current
 profile

Properly dismiss keyguard on boot if the current profile has it
disabled, otherwise we will just show a black screen with just the
status bar and a navbar with only the back button.

BUGBASH-87

Change-Id: I14ed10ae9db40c32be22c1fb43f8a1719787d91b
---
 .../src/com/android/systemui/keyguard/KeyguardViewMediator.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java b/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java
index fdd52b6d4b7..83141f135c9 100644
--- a/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java
+++ b/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java
@@ -1319,6 +1319,8 @@ public class KeyguardViewMediator extends SystemUI {
             if (isKeyguardDisabled(KeyguardUpdateMonitor.getCurrentUser())
                     && !lockedOrMissing && !forceShow) {
                 if (DEBUG) Log.d(TAG, "doKeyguard: not showing because lockscreen is off");
+                setShowingLocked(false);
+                hideLocked();
                 return;
             }
 
-- 
2.17.0

